### PR TITLE
Unit test checkbox

### PR DESCRIPTION
**Pull request by AnisTigrini**
Unit testing for checkbox + some minor improvements.

### CODE REVIEW

File: packages/cypress/cypress/component/Checkbox.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+describe('Checkbox', () => {
+  it('default-notchecked', () => {
+    cy.mount(
+      <Checkbox
+        checked={false}
+        onChange={(checked) => {
+          console.log(checked);
+        }}
+      ></Checkbox>
```
Comment (matthewwolfe) [inline packages/cypress/cypress/component/Checkbox.cy.tsx]: It doesn't matter much but all of these checkboxes can be self closing: `<Checkbox />`

Comment (AnisTigrini) [reply packages/cypress/cypress/component/Checkbox.cy.tsx]: Sure thing, let me modify the PR @matthewwolfe

Comment (AnisTigrini) [reply packages/cypress/cypress/component/Checkbox.cy.tsx]: @matthewwolfe Done :)

Comment (matthewwolfe) [reply packages/cypress/cypress/component/Checkbox.cy.tsx]: Perfect thanks!

**Pull request merged**